From 996a68eb58cadd6dac235254fa67943c84a4b327 Mon Sep 17 00:00:00 2001
From: "Wood, Brian J" <brian.j.wood@intel.com>
Date: Mon, 25 Mar 2019 15:39:59 -0700
Subject: [PATCH 2317/2345] xhci: Correct multiple-line dev_err() statement

There was an extranious comma used in a dev_err()
print function that was causing a compile warning.

Change-Id: I282c9f9b30b4e3ee0e8ea744480fd4bb7b7ac79e
Signed-off-by: Wood, Brian J <brian.j.wood@intel.com>
---
 drivers/usb/host/xhci-ext-caps.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/xhci-ext-caps.c b/drivers/usb/host/xhci-ext-caps.c
index 24835d7..6cefa97 100644
--- a/drivers/usb/host/xhci-ext-caps.c
+++ b/drivers/usb/host/xhci-ext-caps.c
@@ -56,7 +56,7 @@ static int xhci_create_intel_xhci_sw_pdev(struct xhci_hcd *xhci, u32 cap_offset)
 	if (pci->device == PCI_DEVICE_ID_INTEL_CHERRYVIEW_XHCI) {
 		ret = platform_device_add_properties(pdev, role_switch_props);
 		if (ret) {
-			dev_err(dev, "couldn't add properties to",
+			dev_err(dev, "couldn't add properties to"
 					" intel_xhci_usb_sw pdev\n");
 			return ret;
 		}
-- 
2.7.4

